#In the first case, x prints 3

#In the second case, there's an error because x cannot be accessed from the outside